src/main/java/com/puppycrawl/tools/checkstyle/xpath/iterators/ReverseListIterator.java

```diff
@@ -22,7 +22,6 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import com.puppycrawl.tools.checkstyle.xpath.AbstractNode;
```

Comment:
Can we support not usage by import control Check ?

Reply:
Done.